docs/components/Icon/Icon.stories.mdx

```diff
@@ -289,6 +289,7 @@ use the default icon color.
 | <Icon name="eyeCrossed" />   | `eyeCrossed`   |
 | <Icon name="gift" />         | `gift`         |
 | <Icon name="grid" />         | `grid`         |
+| <Icon name="happyFace" />    | `happyFace`    |
```

Comment:
How about `smileyFace`?

Reply:
Thanks for the review, @leticiawanderley !

The labels are coming from the designer (Rebecca L) directly, who changed them from what they [originally were](https://github.com/GetJobber/atlantis/pull/1739/commits/ab9d6f82f2c6ea74df402e57becd69370e039946) based on input from Atlantis. If you feel strongly about it, I can definitely look into getting it changed, but I think since Atlantis suggested them, it might be OK to go with what we have? LMK what you think

Reply:
@rebecca-li I'll leave that call to you

Reply:
I am good with either - I don't have a strong opinion on either naming. Good with changing to `smileyFace` or keeping`happyFace`.